src/main/java/edu/cornell/kfs/module/purap/jaggaer/contract/xml/Contract.java

```diff
@@ -0,0 +1,162 @@
+import jakarta.xml.bind.annotation.XmlType;
+import jakarta.xml.bind.annotation.adapters.XmlJavaTypeAdapter;
+
+@XmlAccessorType(XmlAccessType.FIELD)
+@XmlType(name = "", propOrder = {
+    "contractId",
+    "contractName",
+    "contractNumber",
+    "contractType",
+    "contractStatus",
+    "summary",
+    "contractValue",
+    "contractParties",
+    "customFields",
+    "managerList",
+    "attachments"
+})
+@XmlRootElement(name = "Contract")
+public class Contract {
+
+    @XmlElement(name = "ContractId")
+    private String contractId;
+
+    @XmlElement(name = "ContractName")
+    private String contractName;
+
+    @XmlElement(name = "ContractNumber")
+    private String contractNumber;
+
+    @XmlElement(name = "ContractType")
+    private String contractType;
+
+    @XmlElement(name = "ContractStatus")
+    private String contractStatus;
+
+    @XmlElement(name = "Summary")
+    private String summary;
+
+    @XmlElement(name = "ContractValue")
+    @XmlJavaTypeAdapter(KualiDecimalNullPossibleXmlAdapter.class)
+    private KualiDecimal contractValue;
+
+    @XmlElementWrapper(name = "ContractPartiesList")
+    @XmlElements({
+        @XmlElement(name = "FirstParty", type = FirstParty.class),
+        @XmlElement(name = "SecondParty", type = SecondParty.class)
+    })
+    private List<ContractPartyBase> contractParties;
+
+    @XmlElementWrapper(name = "CustomFieldList")
+    @XmlElement(name = "CustomField")
+    private List<CustomField> customFields;
+
+    @XmlElement(name = "ManagerList")
+    private ManagerList managerList;
+
+    @XmlElementWrapper(name = "AttachmentList")
+    @XmlElement(name = "Attachment")
+    private List<Attachment> attachments;
+
+    public String getContractId() {
+        return contractId;
+    }
+
+    public void setContractId(String contractId) {
+        this.contractId = contractId;
+    }
+
+    public String getContractName() {
+        return contractName;
+    }
+
+    public void setContractName(String contractName) {
+        this.contractName = contractName;
+    }
+
+    public String getContractNumber() {
+        return contractNumber;
+    }
+
+    public void setContractNumber(String contractNumber) {
+        this.contractNumber = contractNumber;
+    }
+
+    public String getContractType() {
+        return contractType;
+    }
+
+    public void setContractType(String contractType) {
+        this.contractType = contractType;
+    }
+
+    public String getContractStatus() {
+        return contractStatus;
+    }
+
+    public void setContractStatus(String contractStatus) {
+        this.contractStatus = contractStatus;
+    }
+
+    public String getSummary() {
+        return summary;
+    }
+
+    public void setSummary(String summary) {
+        this.summary = summary;
+    }
+
+    public KualiDecimal getContractValue() {
+        return contractValue;
+    }
+
+    public void setContractValue(KualiDecimal contractValue) {
+        this.contractValue = contractValue;
```

Comment:
On the the supplier side, the List object getters instantiate the List if it is null.  Should we do that here?

Reply:
I will update this to auto-instantiate the list as needed, as per the discussion at one of our prior team meetings.